packages/ses/test/test-console-error-trap.js

```diff
@@ -34,57 +39,69 @@ const exitAssertions = (t, expectedCode, altExpectedCode = expectedCode) => {
       !stderr.includes('Error#2'),
       'stderr should not contain second error message',
     );
-    t.end();
+    resolve(true);
   };
 };
 
-test.cb('errors reveal their stacks', t => {
+test('errors reveal their stacks', async t => {
```

Comment:
What was `test.cb`?

Reply:
Ava used to have a "callback" helper but they took it out after Ava 3, to use Promises instead